tests/tests_fabric/utilities/test_seed.py

```diff
@@ -30,9 +31,9 @@ def test_seed_stays_same_with_multiple_seed_everything_calls():
         seed_everything()
     initial_seed = os.environ.get("PL_GLOBAL_SEED")
 
-    with pytest.warns(None) as record:
+    with warnings.catch_warnings():
+        warnings.simplefilter("error")
```

Comment:
this seems like just a filter, but the original code validates that no warning was raised...

Reply:
Hi Borda,

According to the [doc](https://docs.python.org/3/library/warnings.html#warning-filter), the `simplefilter("error")`  would turn any warning into an error.

So, when a warning occurs, the test would error out.

Reply:
looks good to me, good use of `catch_warnings` to reset behavior back to default outside the context manager

Reply:
well, your change is equal not to have the `with` context, and we do not text that there are no warnings
```suggestion
```